src/coral_models/prepare_raw_data.py

```diff
@@ -0,0 +1,470 @@
+"""Functions for preparing the raw data"""
+
+import contextlib
+import datetime
+import sqlite3
+import subprocess
+import wave
+from pathlib import Path
+
+import pandas as pd
+import pycountry
+from omegaconf import DictConfig
+from tqdm import tqdm
+
+db_to_excel_metadata_names = {
```

Comment:
Global variables are usually upper case.

```suggestion
DB_TO_EXCEL_METADATA_NAMES = {
```